e3db/types/search.py

```diff
@@ -377,12 +385,27 @@ def range(self, key="CREATED", format="Unix", zone="UTC", zone_offset=None, star
             Search only for records that come before this time 
             (the default is None, which leaves no upper bound on the query)
         
+        zone_offset : int, optional
+            Accepts int for provided timezone in hour difference from UTC.
+            For PST(UTC-8) provide zone_offset = -8
+            For PDT(UTC-7) provide zone_offset = -7
```

Comment:
need to add either the remaining timezone int codes here or document it elsewhere

Reply:
Any hour offset from utc should be a viable input, but I will provide more comprehensive answers here and in the readme